packages/fxa-payments-server/src/routes/Subscriptions/PaymentUpdateForm.tsx

```diff
@@ -125,6 +125,22 @@ export const PaymentUpdateForm = ({
         </div>
       ) : (
         <>
+          <ErrorMessage isVisible={!!createTokenError.error}>
```

Comment:
I like `isVisible` being a Boolean value since it's more specific, but I know there's a gotcha around using `!!` if this is ever an empty string (it's not in this case) and some folks just don't like it in general. I liked it over `new Boolean(createTokenError.error)` or passing in the entire object + taking in an `any` type for `isVisible` 🤷‍♀️